custom-options-plus.php

```diff
@@ -44,10 +44,21 @@
 
 global $wpdb, $COP_TABLE;
 define( 'COP_TABLE', $wpdb->prefix . 'custom_options_plus' );
+define('COP_SECURITY_STRING', date('h'));
```

Comment:
In my view, an ajax nonce is a param that you pass in ajax request to garantee it is really valid, but this can be broken if you know the ajax nonce and it keeps the same all the time, the solution is change it every hour (that's why used date('h') for it, because it changes every hour), to keep the ajax nonce sync with the client(if you don't reload the page for a hour, the server will generate another nonce and client will keep with the old one), so i done a refresh control to reload the page if a hour passes, well, it should be improved to be real secure, but it's a start.